src/cli/commands/files/add.js

```diff
@@ -1,11 +1,16 @@
 'use strict'
 
 const Command = require('ronin').Command
-const IPFS = require('../../../core')
+const utils = require('../../utils')
 const debug = require('debug')
 const log = debug('cli:version')
 log.error = debug('cli:version:error')
 const bs58 = require('bs58')
+const Readable = require('stream').Readable
+const fs = require('fs')
+const async = require('async')
+const pathj = require('path')
```

Comment:
use `path` for the name of the variable that contains the module, and use another keyword for the argument path. So that we keep the usage of the module consistent